Client/Frontend/Home/Wallpapers/WallpaperDataManager.swift

```diff
@@ -112,22 +123,31 @@ struct WallpaperDataManager {
     }
     
     private func firefoxDefaultCollection() -> [WallpaperCollection] {
-        return [WallpaperCollection(wallpaperFileNames: ["fxCerulean",
-                                                         "fxAmethyst",
-                                                         "fxSunrise"],
-                                    ofType: .themed(type: .firefox))]
+        return [WallpaperCollection(
+            wallpaperFileNames: [WallpaperID(name: "fxSunrise",
+                                             accessibilityID: accessibilityIDs.FxSunriseWallpaper)],
+            ofType: .themed(type: .firefox)),
+                WallpaperCollection(
+            wallpaperFileNames: [WallpaperID(name: "fxCerulean",
+                                             accessibilityID: accessibilityIDs.FxCeruleanWallpaper),
+                                 WallpaperID(name: "fxAmethyst",
+                                             accessibilityID: accessibilityIDs.FxAmethystWallpaper)],
+            ofType: .themed(type: .firefoxOverlay))]
     }
     
     private func allSpecialCollections() -> [WallpaperCollection]? {
         var specialCollections = [WallpaperCollection]()
 
         let houseExpiryDate = Calendar.current.date(
             from: DateComponents(year: 2022, month: 5, day:1))
-        let projectHouse = WallpaperCollection(wallpaperFileNames: ["trRed",
-                                                                    "trGroup"],
-                                               ofType: .themed(type: .projectHouse),
-                                               expiringOn: houseExpiryDate,
-                                               limitedToLocales: ["en_US", "es_US"])
+        let projectHouse = WallpaperCollection(
+            wallpaperFileNames: [WallpaperID(name: "trRed",
+                                             accessibilityID: "Turning Red wallpaper, giant red panda, button"),
```

Comment:
No need to add "button" at the end of the string. This is already part of voice over if the wallpaper tile has the button trait. I don't see this at the moment, so I guess you have to add it with `.accessibilityTraits = .button` on the wallpaper tiles.

I think you'll also need to put the trait "user interaction enabled" on your wallpaper tiles. Those are the traits that are there by default on a UIButton. Since the wallpaper tile reenacts a button I think it should have the same traits.
![Screen Shot 2022-02-16 at 9 53 57 AM](https://user-images.githubusercontent.com/11338480/154290620-df169ffb-7ea4-4dd6-802b-4a7d2c074684.png)

Reply:
So, the wallpaper cells are not buttons. I just treated them as selectable cells instead of doing the whole button dance. This is why I added button to the strings.